mobile/src/screens/profile-settings/profile-settings.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import {
+  Header,
+  LinearGradient,
+  ScrollView,
+  Switch,
+} from '#libs/components/components';
+import { useAppForm } from '#libs/hooks/hooks';
+import { type NavigationScreenProperties } from '#libs/types/types';
+
+import { Setting } from './components/components';
+import { NOTIFICATION_SETTINGS_DEFAULT_VALUE } from './libs/constants';
+import { styles } from './styles';
+
+const ProfileSettings = ({
+  route,
+}: NavigationScreenProperties): JSX.Element => {
```

Comment:
Uer React.FC
Also, we don't need `route` here. Header component uses route name by default